cpp/libs/gst_nmos_sender_plugin/gst_nmos_sender_plugin.cpp

```diff
@@ -243,27 +311,44 @@ static GstStateChangeReturn gst_nmossender_change_state(GstElement* element, Gst
 
     switch(transition)
     {
-    case GST_STATE_CHANGE_NULL_TO_READY: {
+    case GST_STATE_CHANGE_PAUSED_TO_PLAYING: {
 
-        const auto node_config_json   = create_node_config(self->config);
-        const auto device_config_json = create_device_config(self->config);
-        const auto sender_config_json = create_sender_config(self->config);
+        const auto node_config_json                         = create_node_config(self->config);
+        const auto device_config_json                       = create_device_config(self->config);
+        nlohmann::json_abi_v3_11_3::json sender_config_json = NULL;
```

Comment:
Avoid `NULL` and use `nullptr`.
Avoid `TRUE`/`FALSE` in `bool` variables and use `true`/`false`.